src/main/java/io/reactivex/schedulers/TestScheduler.java

```diff
@@ -23,9 +23,60 @@
 import io.reactivex.internal.functions.ObjectHelper;
 
 /**
- * A special, non thread-safe scheduler for testing operators that require
- * a scheduler without introducing real concurrency and allows manually advancing
- * a virtual time.
+ * This special {@link Scheduler} allows testing asynchronous and/or timed operators
+ * and flows by programatically executing the scheduled tasks up to a certain virtual
+ * time, allowing checking for intermediate results, state changes or preparing the relevant
+ * context (global/captured object state) before executing the next set of tasks.
+ * <p><p>
```

Comment:
why `<p>` twice?

Reply:
Seems like we don't get a blank line otherwise.